### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by lance6716**
1. replace Chinese comma
2. unify index-key format

### CODE REVIEW

File: tidb-internal-2.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Key: tablePrefix{tableID}_indexPrefixSep_indexID_indexedColumnsValue
 Value: rowID
 ```
 
-Index 数据还需要考虑 Unique Index 和非 Unique Index 两种情况，对于 Unique Index，可以按照上述编码规则。但是对于非 Unique Index，通过这种编码并不能构造出唯一的 Key，因为同一个 Index 的 `tablePrefix_tableID_indexPrefixSep_indexID` 都一样，可能有多行数据的 `ColumnsValue` 是一样的，所以对于非 Unique Index 的编码做了一点调整：
+Index 数据还需要考虑 Unique Index 和非 Unique Index 两种情况，对于 Unique Index，可以按照上述编码规则。但是对于非 Unique Index，通过这种编码并不能构造出唯一的 Key，因为同一个 Index 的 `tablePrefix{tableID}_indexPrefixSep_indexID` 都一样，可能有多行数据的 `ColumnsValue` 是一样的，所以对于非 Unique Index 的编码做了一点调整：
```
Comment (kennytm) [inline tidb-internal-2.md]: If we're changing this, the `indexID` should also be treated the same i.e. `tablePrefix{tableID}_indexPrefixSep{indexID}`. Lines 63 and 70 also need to be updated.

**Comment by kennytm** [comment]
PTAL @CaitinChen

**Comment by CaitinChen** [comment]
@husiyu PTAL

File: tidb-internal-2.md (hunk 1 above)
Comment (lance6716) [reply tidb-internal-2.md]: I agree, now I prefer these string formatted as line 54

**Comment by lance6716** [comment]
`prefix_ID` now become `prefix{ID}`, so lines 54, 63, 67, 70 looks the same style. @CaitinChen

**Comment by shenli** [comment]
LGTM

**Pull request merged**